aurora-postgres.tf

```
@@ -0,0 +1,188 @@
+variable "postgres_name" {
+  type        = "string"
+  description = "Name of the application, e.g. `app` or `analytics`"
+  default     = ""
+}
+
+# Don't use `admin` 
+# Read more: <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html>
+# ("MasterUsername admin cannot be used as it is a reserved word used by the engine")
+variable "postgres_admin_user" {
+  type        = "string"
+  description = "Postgres admin user name"
+  default     = ""
+}
+
+# Must be longer than 8 chars
+# Read more: <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html>
+# ("The parameter MasterUserPassword is not a valid password because it is shorter than 8 characters")
+variable "postgres_admin_password" {
+  type        = "string"
+  description = "Postgres password for the admin user"
+  default     = ""
+}
+
+variable "postgres_db_name" {
+  type        = "string"
+  description = "Postgres database name"
+  default     = ""
+}
+
+# db.r4.large is the smallest instance type supported by Aurora Postgres
+# https://aws.amazon.com/rds/aurora/pricing
+variable "postgres_instance_type" {
+  type        = "string"
+  default     = "db.r4.large"
+  description = "EC2 instance type for Postgres cluster"
+}
+
+variable "postgres_cluster_size" {
+  type        = "string"
+  default     = "2"
+  description = "Postgres cluster size"
+}
+
+variable "postgres_cluster_enabled" {
+  type        = "string"
+  default     = "true"
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "postgres_cluster_family" {
+  type        = "string"
+  default     = "aurora-postgresql9.6"
+  description = "Postgres cluster DB family. Currently supported values are aurora-postgresql9.6 / aurora-postgresql10"
+}
+
+variable "postgres_maintenance_window" {
+  type        = "string"
+  default     = "sun:03:00-sun:04:00"
+  description = "Weekly time range during which system maintenance can occur, in UTC"
+}
+
+locals {
+  postgres_cluster_enabled = "${var.postgres_cluster_enabled == "true"}"
+  postgres_admin_user      = "${length(var.postgres_admin_user) > 0 ? var.postgres_admin_user : join("", random_string.postgres_admin_user.*.result)}"
+  postgres_admin_password  = "${length(var.postgres_admin_password) > 0 ? var.postgres_admin_password : join("", random_string.postgres_admin_password.*.result)}"
+  postgres_db_name         = "${join("", random_pet.postgres_db_name.*.id)}"
+  postgres_name            = "${length(var.postgres_name) > 0 ? var.postgres_name : module.label.id}"
+}
+
+module "aurora_postgres" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=tags/0.10.0"
+  namespace          = "${var.namespace}"
+  stage              = "${var.stage}"
+  name               = "${local.postgres_name}"
```

```suggestion
  name               = "${lvar.name}"
```

Should be just `var.name` since `label.id` is generated inside the module `terraform-aws-rds-cluster`